packages/clay-icon/src/ClayIcon.js

```diff
@@ -25,6 +25,15 @@ ClayIcon.STATE = {
 	 */
 	elementClasses: Config.string(),
 
+	/**
+	 * Flag to indicate when svg focusable or not.
```

Comment:
Typo: Flag to indicate if the svg is focusable or not

Reply:
That looks better, thanks!